tests/local_contract_calls.py

```diff
@@ -85,6 +95,7 @@ class ReceiptAggregateVoucher(EIP712Message):
     print("Approving escrow contracts")
     erc20.functions.approve(ESCROW_ADDRESS, 10000).transact({"from": GATEWAY})
     erc20.functions.approve(ESCROW_ADDRESS, 10000).transact({"from": SIGNER})
+    erc20.functions.approve(ESCROW_ADDRESS, 10000).transact({"from": SIGNER})
```

Comment:
Is it normal that it's the same line twice?

Reply:
yeah no, added it for a test forgot to remove it